_episodes/03-packaging-installing.md

```diff
@@ -116,129 +117,133 @@ This shows the basic commands and the general options.
 
 ## Python Modules
 
-A module is a piece of code that serves a specific purpose. In python, a module is written in a `.py` file and the name of the file is name of the module. A module can contain classes, functions, or a combination of both. Modules can also define variables for use, for example, `numpy` defines the value of pi (`numpy.Pi`).
+A module is a piece of code that serves a specific purpose. In python, a module is written in a `.py` file. The name of the file is name of the module. A module can contain classes, functions, or a combination of both. Modules can also define variables for use, for example, [numpy](https://numpy.org/) defines the value of pi with `numpy.pi`.
+
+
+If a `.py` file is on the path, we can import functions from it to our current file. Open up Python, import `sys` and print the path.
+
+~~~
+import sys
+sys.path
+~~~
+{:.language-python}
 
+sys.path is a list of strings, each describing the absolute path to a directory. Python will look in these directories for modules. If we have a directory containing modules we want Python to be aware of, we append it to the path:
 
-If the `.py` file is on the path, we can import functions from one file to another.
 
-FIXME: example
+~~~
+sys.path.append('/path/to/module/')
+sys.path
+~~~
+{:.language-python}
 
+Once the module you want is in sys.path, it can be imported just like any other module.
 
 ## Python Packages
 
-We don't have to manually add to the path every time though. Instead, we can
-package our modules to be installable, and then we can import directly.  This
-way of importing allows us to better allow python on the backend handle differences in paths across different user systems.  
+To save adding modules to the path every time we want to use them, we can
+package our modules to be installable.  This
+method of importing standardises how we import modules across different user systems.
 
-Packages are namespaces or containers (e.g. a directory) which can contain multiple packages and modules.
+Packages are namespaces or containers which can contain multiple modules. e.g. the `scipy` package contains the `interpolate` and `optimize` modules.
```

Comment:
we should define namespace.  I don;t think the target audience knows that

Reply:
Added in a27b164